src/main/java/com/fivetran/external/com/amazonaws/services/kinesis/leases/impl/LeaseFetchOrder.java

```diff
@@ -0,0 +1,6 @@
+package com.fivetran.external.com.amazonaws.services.kinesis.leases.impl;
+
+public enum LeaseFetchOrder {
+    RANDOM,
```

Comment:
Out of curiosity - Once we roll out the `PARENTS_FIRST` fetch order is there any benefit to keeping the RANDOM fetch order?

Reply:
Yes. This Worker was designed to be used in a multi-Worker setup where they all are pulling from the same pool of leases. Having them select from the pool randomly reduces the chances of collision. You can read more about the intended setup here: https://aws.amazon.com/blogs/big-data/process-large-dynamodb-streams-using-multiple-amazon-kinesis-client-library-kcl-workers/

I want to preserve this functionality for two reasons:
1. Make sure it is available to us if we dramatically rethink how to use the dynamo connector and use multiple workers per table
2. This repo is open-source and I felt like if someone else may want to have both options available.